birkbeck/db1/README.md

```diff
@@ -2,63 +2,76 @@
-
-## Ever quickest intro to CLI
-
-Command Line Interface (or CLI), is the eldest interface of all. The mighty command line or terminal, stemming from the physical terminals of the 70's (e.g. insert ANYTHING here sometimes without even a display).
+## A (very) brief introduction to command-line interfaces
 
+The Command Line Interface (or CLI) originated in mid-1960s computer terminals like the IBM machine in the image below. These were modelled on typewriters and did not always include display monitors, instead featuring just a keyboard and printer. Technological limitations forced the adoption of a text-based system for navigating filing systems.
 
 ![an old terminal](https://upload.wikimedia.org/wikipedia/commons/thumb/2/2c/IBM2741.JPG/1280px-IBM2741.JPG)
 
+Today, it is the norm and not the exception for a computer to include a display. The advent of graphical user interfaces revolutionised the way computers were used and extended their appeal beyond secretarial and accounting departments. Command-line interfaces are still packaged with every UNIX-based system (e.g., OSX, Linux) and can be emulated in Windows, but given the demands they place on our memory and concentration, why should we bother?
+
+I don't think graphical user interfaces really improved on the tasks that computers were first used for. Yes, you can effortlessly browse Facebook on a MacBook Pro, but how quickly could you list the contents of your hard drive? Where would you go to find out your IP address? The tasks that computing professionals typically do require the command line. Hopefully you will see that is an elegant solution to many common problems in bioinformatics.
 
-Luckily enough today most laptops ship with a display :D CLI can be used to access the filesystem, installed programs in the remote machine (like editors or databases e.tc.) or even connected peripherals (like a printer).
+Command-line interfaces respond to commands. These are short words which may sometimes be followed by a file path or optional characters to specify additional details. Here are a few of the most popular and useful command-line commands:
 
-![a screen terminal](https://upload.wikimedia.org/wikipedia/commons/8/87/Televideo925Terminal.jpg)
+- `whoami`
+No, this isn't a philosophy lesson. This command simply returns the username you are currently using to access the system. This can be useful to check you have logged in correctly.
 
-The most used and useful commands for CLI are:
-- `whoami` (sounds philosophical but it is helpful)
 - `pwd` (Print Working Directory)
+In other words, where am I? This command tells you which directory (or folder) you are currently operating in. This is important to know, because actions taken which do not specify an explicit alternate directory will result in changes being made to the directory you are currently in. Use this to make sure you're in the right place when deleting old files, and not accidentally deleting the contents of your coursework folder.
```

Comment:
I find this a bit wordy. I wanted to keep the Unix intro as short as possible, so I would say to only keep the first two sentences; i.e. until "...operating in.".

Reply:
Yep, fair!